lib/mayaUsd/fileio/utils/meshReadUtils.cpp

```diff
@@ -176,7 +177,26 @@ MIntArray getMayaFaceVertexAssignmentIds(
         } else if (interpolation == UsdGeomTokens->vertex) {
             valueId = itFV.vertId();
         } else if (interpolation == UsdGeomTokens->faceVarying) {
-            valueId = fvi;
+            if (isLeftHanded) {
+                // When the mesh is left-handed, face winding order was reversed.
+                // We need to adjust the face-vertex index to match the corrected order.
+                // Find the position of this vertex within its face and reverse it.
+                const int faceId = itFV.faceId();
+                int vertexCount = meshFn.polygonVertexCount(faceId);
+
+                // Find which vertex we are within this face
+                unsigned int baseFvi = 0;
+                for (int f = 0; f < faceId; ++f) {
+                    int faceVertCount = meshFn.polygonVertexCount(f);
+                    baseFvi += faceVertCount;
+                }
+
+                const int vertexPosInFace = fvi - baseFvi;
+                const int reversedVertexPosInFace = vertexCount - 1 - vertexPosInFace;
+                valueId = baseFvi + reversedVertexPosInFace;
+            } else {
+                valueId = fvi;
+            }
```

Comment:
Handle UVs for left-handed meshes. See #1139 where a change was made to support faces correctly for left-handled meshes. That PR fixed that problem but exposed this problem with the UVs.